datafusion/sqllogictest/test_files/string/string_query.slt.part

```diff
@@ -694,3 +694,287 @@ Andrew nice Andrew and X datafusion📊🔥 cool datafusion📊🔥 and 🔥 And
+             Raphael datafusionДатаФусион
+NULL NULL
+
+# --------------------------------------
+# Test RPAD
+# --------------------------------------
+
+query TTTT
+SELECT
+  RPAD(ascii_1, 20, 'x'),
+  RPAD(ascii_1, 20, NULL),
+  RPAD(unicode_1, 20, '🔥'),
+  RPAD(unicode_1, 20, NULL)
+FROM test_basic_operator;
+----
+Andrewxxxxxxxxxxxxxx NULL datafusion📊🔥🔥🔥🔥🔥🔥🔥🔥🔥 NULL
+Xiangpengxxxxxxxxxxx NULL datafusion数据融合🔥🔥🔥🔥🔥🔥 NULL
+Raphaelxxxxxxxxxxxxx NULL datafusionДатаФусион NULL
+NULL NULL NULL NULL
+
+query TT
+SELECT
+  RPAD(ascii_1, 20),
+  RPAD(unicode_1, 20)
+FROM test_basic_operator;
+----
+Andrew              datafusion📊🔥
+Xiangpeng           datafusion数据融合
+Raphael             datafusionДатаФусион
+NULL NULL
+
+# --------------------------------------
+# Test REGEXP_LIKE
+# --------------------------------------
+
+# TODO: LargeString does not support REGEXP_LIKE. Enable this after fixing the issue
+# see issue: https://github.com/apache/datafusion/issues/12664
+#query BBBB
+#SELECT
+#  REGEXP_LIKE(ascii_1, 'an'),
+#  REGEXP_LIKE(unicode_1, 'таФ'),
+#  REGEXP_LIKE(ascii_1, NULL),
+#  REGEXP_LIKE(unicode_1, NULL)
+#FROM test_basic_operator;
+#----
+#false false NULL NULL
+#true false NULL NULL
+#false true NULL NULL
+#NULL NULL NULL NULL
+
+# --------------------------------------
+# Test REGEXP_MATCH
+# --------------------------------------
+
+# TODO: LargeString does not support REGEXP_MATCH. Enable this after fixing the issue
+# see issue: https://github.com/apache/datafusion/issues/12664
+#query ????
+#SELECT
+#  REGEXP_MATCH(ascii_1, 'an'),
+#  REGEXP_MATCH(unicode_1, 'таФ'),
+#  REGEXP_MATCH(ascii_1, NULL),
+#  REGEXP_MATCH(unicode_1, NULL)
+#FROM test_basic_operator;
+#----
+#NULL NULL NULL NULL
+#[an] NULL NULL NULL
+#NULL [таФ] NULL NULL
+#NULL NULL NULL NULL
+
+# --------------------------------------
+# Test REPEAT
+# --------------------------------------
+
+query TT
+SELECT
+  REPEAT(ascii_1, 3),
+  REPEAT(unicode_1, 3)
+FROM test_basic_operator;
+----
+AndrewAndrewAndrew datafusion📊🔥datafusion📊🔥datafusion📊🔥
+XiangpengXiangpengXiangpeng datafusion数据融合datafusion数据融合datafusion数据融合
+RaphaelRaphaelRaphael datafusionДатаФусионdatafusionДатаФусионdatafusionДатаФусион
+NULL NULL
+
+# --------------------------------------
+# Test SPLIT_PART
+# --------------------------------------
+
+query TTTTTT
+SELECT
+  SPLIT_PART(ascii_1, 'e', 1),
+  SPLIT_PART(ascii_1, 'e', 2),
+  SPLIT_PART(ascii_1, NULL, 1),
+  SPLIT_PART(unicode_1, 'и', 1),
+  SPLIT_PART(unicode_1, 'и', 2),
+  SPLIT_PART(unicode_1, NULL, 1)
+FROM test_basic_operator;
+----
+Andr w NULL datafusion📊🔥 (empty) NULL
+Xiangp ng NULL datafusion数据融合 (empty) NULL
+Rapha l NULL datafusionДатаФус он NULL
+NULL NULL NULL NULL NULL NULL
+
+# --------------------------------------
+# Test REVERSE
+# --------------------------------------
+
+query TT
+SELECT
+  REVERSE(ascii_1),
+  REVERSE(unicode_1)
+FROM test_basic_operator;
+----
+werdnA 🔥📊noisufatad
```

Comment:
Werdna -- this brings back memories from my childhood when sometimes kids would refer to my name like this.